circle.yml

```diff
@@ -0,0 +1,9 @@
+test:
+  override:
+    - "true"
+deployment:
+  master:
+    branch: master
+    commands:
+      - npm install
```

Comment:
This should be done automatically through inferences. Are you sure you need it, especially at the `deployment` stage?